osu.Game.Tournament/Screens/SetupScreen.cs

```diff
@@ -186,15 +189,70 @@ public bool Failing
                         Anchor = Anchor.CentreLeft,
                         Origin = Anchor.CentreLeft,
                     },
-                    button = new TriangleButton
+                    FlowContainer = new FillFlowContainer
                     {
                         Anchor = Anchor.CentreRight,
                         Origin = Anchor.CentreRight,
-                        Size = new Vector2(100, 30),
-                        Action = () => Action?.Invoke()
-                    },
+                        AutoSizeAxes = Axes.Both,
+                        Spacing = new Vector2(10, 0),
+                        Children = new Drawable[]
+                        {
+                            Button = new TriangleButton
+                            {
+                                Size = new Vector2(100, 30),
+                                Action = () => Action?.Invoke()
+                            }
+                        }
+                    }
                 }
             };
         }
+
+        private class ResolutionSelector : ActionableInfo
+        {
+            private const int height_min_allowed_value = 480;
+            private const int height_max_allowed_value = 2160; // 4k
+            public new Action<int> Action;
+
+            private OsuNumberBox numberBox;
+
+            protected override Drawable CreateComponent()
+            {
+                var drawable = base.CreateComponent();
+                FlowContainer.Insert(0, numberBox = new OsuNumberBox
+                {
+                    Width = 100
+                });
+                FlowContainer.SetLayoutPosition(Button, 1);
```

Comment:
This is my oversight - the `0` doesn't work as the button already has layout position 0. `-1` will work (it's not a conventional list).

Reply:
oooh right so its basically just arbitrary numbers and in the end the elements are ordered by whatever values they have?

Reply:
Yes. `Insert` internally calls `SetLayoutPosition`.